controllers/account.js

```diff
@@ -55,6 +57,7 @@ module.exports.renderChangeProfile = async (req, res) => {
 };
 
 module.exports.updatePassword = async (req, res) => {
+  dataSchemaValidation(req.body, changePasswordSchema);
```

Comment:
In other places you put an if check for `!error` value returned from validator, why not here as well? Although it does not matter much if you're planning to refactor it to middleware